pipelines/api-pr-pipeline.yml

```diff
@@ -100,12 +100,16 @@ stages:
                     Import-Module FusionPS
 
                     $dbName = "Fusion-Apps-Resources-PR-$(prNumber)-DB"
+                    $tags = @{ 
+                        pr = "$(prNumber)"
+                        fusion-app-component-id = "resources-api-db-pr-$(prNumber)"
```

Comment:
must quote properties.. - is not a valid param name by default.

Reply:
Yup, added kvotes. PS skills not impressive :D